docs/additional_samples.rst

```diff
@@ -209,7 +209,7 @@ Math
 
    .. math::
 
-      (a + b)^2 = a^2 + 2ab + b^2
+      (a + b)^2 = a^2 + 2ab + b^2 \\
```

Comment:
It might be worth noting when the escaped line breaks are needed/useful in a doc dedicated to this theme-ext